one_fm/grd/doctype/moi_residency_jawazat/test_moi_residency_jawazat.py

```diff
@@ -6,6 +6,8 @@
 from frappe.utils import today, add_days, get_url
 import unittest
 
+
```

Comment:
Remove the whitespaces and undone these changes

Reply:
done..